### PR TITLE
Add exception message on clearing running connections from pool

### PR DESCRIPTION
**Pull request by zabulus**

### CODE REVIEW

File: NETProvider/src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbConnectionPoolManager.cs [hunk 1]
```diff
@@ -184,8 +184,10 @@ void CleanConnectionsImpl()
 					},
 					() =>
 					{
-						foreach (var item in _busy)
-							item.Dispose();
+						if(_busy.Any())
```
Comment (cincuranet) [inline NETProvider/src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbConnectionPoolManager.cs]: Wrong formatting.

File: NETProvider/src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbConnectionPoolManager.cs [hunk 2]
```diff
@@ -184,8 +184,10 @@ void CleanConnectionsImpl()
...
+						{
+							throw new InvalidOperationException("Running connections exist.");
```
Comment (cincuranet) [inline NETProvider/src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbConnectionPoolManager.cs]: "Running"? What's that?

File: NETProvider/src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbConnectionPoolManager.cs (hunk 1 above)
Comment (cincuranet) [inline NETProvider/src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbConnectionPoolManager.cs]: Shouldn't there be also an `else` clause?

**Comment by cincuranet** [comment]
Don't forget to put it into tracker as I told you in list.

File: NETProvider/src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbConnectionPoolManager.cs (hunk 1 above)
Comment (zabulus) [inline NETProvider/src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbConnectionPoolManager.cs]: What there should be?

File: NETProvider/src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbConnectionPoolManager.cs (hunk 2 above)
Comment (zabulus) [inline NETProvider/src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbConnectionPoolManager.cs]: I'll fix that, just added the PR, to check test runs

**Comment by zabulus** [comment]
> Don't forget to put it into tracker as I told you in list.
http://tracker.firebirdsql.org/browse/DNET-638

I'm going to add tests that expect the exception. And fix the exception message

**Comment by cincuranet** [comment]
Reading the documentation for `SqlClient`:

> ClearAllPools resets (or empties) the connection pool. If there are connections in use at the time of the call, they are marked appropriately and will be discarded (instead of being returned to the pool) when Close is called on them.

Seems to be like an interesting idea.

**Pull request closed without merging**